pkg/cloudprovider/vsphereparavirtual/cloud.go

```diff
@@ -29,13 +29,16 @@ import (
 
 	cpcfg "k8s.io/cloud-provider-vsphere/pkg/common/config"
 	k8s "k8s.io/cloud-provider-vsphere/pkg/common/kubernetes"
+	"k8s.io/cloud-provider-vsphere/pkg/common/provider"
 )
 
 const (
-	// ProviderName is the name of the cloud provider registered with
-	// Kubernetes.
-	ProviderName string = "vsphere-paravirtual"
-	clientName   string = "vsphere-paravirtual-cloud-controller-manager"
+	// ProviderNameForRegister is only uesed when register CP.
+	// https://github.com/kubernetes/cloud-provider/blob/03b2e3fe9feb25c0620c23fbc3bf43925e377a9c/plugins.go#L55
+	// To distinguish from vsphere CP, use different name
+	ProviderNameForRegister string = "vsphere-paravirtual"
```

Comment:
I'm thinking that here we can just have:
```
ProviderName string = "vsphere-paravirtual"
```

And just hardcode `ProviderName()` below to return "vsphere". What do you think?

Reply:
Ok, that was previous version. Let me change it back